tokens/src/core/global/typography.json

```diff
@@ -2,89 +2,222 @@
   "typography": {
     "font": {
       "family": {
+        "$type": "fontFamily",
         "base": {
-          "value": "{typography.font.family.sans.serif}",
-          "type": "fontFamily",
           "source": "$font-family-base",
-          "description": "Basic font family."
+          "$value": "{typography.font.family.sans.serif}",
+          "$description": "Basic font family."
         },
         "sans": {
           "serif": {
-            "value": "-apple-system, BlinkMacSystemFont, \"Segoe UI\",  Roboto, \"Helvetica Neue\", Arial, \"Noto Sans\", sans-serif, \"Apple Color Emoji\",  \"Segoe UI Emoji\", \"Segoe UI Symbol\", \"Noto Color Emoji\"",
-            "type": "fontFamily",
             "source": "$font-family-sans-serif",
-            "description": "Sans-serif font family."
+            "$value": "-apple-system, BlinkMacSystemFont, \"Segoe UI\",  Roboto, \"Helvetica Neue\", Arial, \"Noto Sans\", sans-serif, \"Apple Color Emoji\",  \"Segoe UI Emoji\", \"Segoe UI Symbol\", \"Noto Color Emoji\"",
+            "$description": "Sans-serif font family."
           }
         },
-        "serif": { "value": "serif", "source": "$font-family-serif", "description": "Serif font family." },
+        "serif": {
+          "source": "$font-family-serif",
+          "$value": "serif",
+          "$description": "Serif font family."
+        },
         "monospace": {
-          "value": "SFMono-Regular, Menlo, Monaco, Consolas, \"Liberation Mono\",  \"Courier New\", monospace",
-          "type": "fontFamily",
           "source": "$font-family-monospace",
-          "description": "Monospace font family."
+          "$value": "SFMono-Regular, Menlo, Monaco, Consolas, \"Liberation Mono\",  \"Courier New\", monospace",
+          "$description": "Monospace font family."
         }
       },
       "size": {
-        "base": { "value": "1.125rem", "type": "dimension", "source": "$font-size-base", "description": "Base font size." },
-        "lg": { "value": "1.4063rem", "type": "dimension", "source": "$font-size-lg", "description": "Large font size." },
-        "sm": { "value": ".875rem", "type": "dimension", "source": "$font-size-sm", "description": "Small font size." },
-        "xs": { "value": ".75rem", "type": "dimension", "source": "$font-size-xs", "description": "Extra-small font size." },
+        "base": {
+          "source": "$font-size-base",
+          "$value": "1.125rem",
+          "$type": "dimension",
```

Comment:
[suggestion] could the `"$type": "dimension"`, be moved up under `size` so it applies to all size permutations (e.g., base vs. lg vs. sm)?

Reply:
[inform] Implemented in follow-up PR: https://github.com/openedx/paragon/pull/3203